decentraland/unity-explorer
# style: log in images update

## What does this PR change?

This PR contains the new screenshots for the log in screen. Video attached bellow.
https://github.com/decentraland/unity-explorer/assets/51088292/a85b6963-43d6-45eb-92fc-b0c4e420a23b

## How to test the changes?

1. Launch the explorer
2. If you land in the lobby screen, select the option to log in with another account. If you land in the log in screen check the  screenshots are the new ones.
